### PR TITLE
Single account multiple workers

### PR DESCRIPTION
**Pull request by bitfag**
Handle multiple workers on same account with Staggered Orders. The feature was funded by Cryptobridge.

### CODE REVIEW

**Comment by joelvai** [comment]
To test this properly on GUI as well, you need to remove validation from `worker_controller.py` which doesn't allow creation of another worker with same account.

**Comment by thehapax** [comment]
@PermieBTS requesting additional end-users to test single-account-multiple workers to ensure no-err on funds.

**Comment by PermieBTS** [comment]
I'm looking into ways that our DEXBot community users can help us test.
The problem users give is that they would have to stop their main-workers which they want to run 24/7... in order to test a new feature/version.

So easy ways to run multiple DEXBot instances are needed.

On Linux, if one creates two users, then one user can run DEXBot silently in the back ground and the other can run DEXBot too.

1. Create a new user
2. Install desired DEXBot version on that user
3. Log in to DEXBot, configure settings, and run once.
4. After this you can start and run this DEXBot worker in a terminal

su otheruser *changes to other user*
screen dexbot-cli run *screen lets you detach change user without stopping the bot*
ctrl+a and ctrl+d *changes screen to control mode*
ctrl+d *is exit*

*To reattach and control the other bot again:*
su otheruser
screen -r


====================
The DEXBot docker image can also be used to run two simultaneous versions of DEXBot

**Comment by bitfag** [comment]
You may ask cryptobridge team as this feature was implemented by their request and they are active users of it.

**Comment by bitfag** [comment]
>joelva2:43 PM
>@vvk the multiple workers on one account breaks when closing the second one,

```
Traceback (most recent call last):
  File "/home/joel/projects/DEXBot/dexbot/views/errors.py", line 95, in func_wrapper
    return func(*args, **kwargs)
  File "/home/joel/projects/DEXBot/dexbot/views/worker_item.py", line 87, in pause_worker
    self.main_ctrl.pause_worker(self.worker_name)
  File "/home/joel/projects/DEXBot/dexbot/controllers/main_controller.py", line 58, in pause_worker
    self.worker_manager.stop(worker_name, pause=True)
  File "/home/joel/projects/DEXBot/dexbot/worker.py", line 198, in stop
    self.accounts.remove(account)
KeyError: 'test-account'
```

**Comment by PermieBTS** [comment]
Cryptokong is recieving training on how to test this strategy and recruit end-users to test this strategy

File: dexbot/strategies/base.py [hunk 1]
```diff
@@ -158,7 +158,11 @@ def configure(cls, return_base_config=True):
                           r'[A-Z0-9\.]+[:\/][A-Z0-9\.]+'),
             ConfigElement('fee_asset', 'string', 'BTS', 'Fee asset',
                           'Asset to be used to pay transaction fees',
-                          r'[A-Z\.]+')
+                          r'[A-Z\.]+'),
+            ConfigElement('operational_percent_quote', 'float', 0, 'QUOTE balance %',
+                          'Max % of QUOTE asset available to this worker', (0, None, 2, '%')),
+            ConfigElement('operational_percent_base', 'float', 0, 'BASE balance %',
+                          'Max % of BASE asset available to this worker', (0, None, 2, '%')),
```
[the remark this answers is not in the document]
Comment (bitfag) [reply dexbot/strategies/base.py]: Ok, added.

**Comment by bitfag** [comment]
@joelvai can you describe in more details of how you got error `KeyError: 'test-account'`? I'm unable to reproduce it.

**Comment by joelvai** [comment]
@bitfag sorry for delayed answer. 
1. Create two workers that have same account
2. Remove worker 2
3. Remove worker 3

**Comment by PermieBTS** [comment]
Merging of this feature is delayed until unit tested

**Comment by bitfag** [comment]
To obtain win executable for testing, go to "continuous-integration/appveyor/pr" Details -> Artifacts -> download DEXBot-gui-win64.zip

**Pull request merged**